lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
```

Comment:
so will this only execute if the cc yaml looks something like
```
prepare:
  quality: true
```
?  is that something to cover in a spec?